source/enumerate-databases.rst

```diff
@@ -134,6 +157,8 @@ Older versions of the server that do not support the ``nameOnly`` option for the
 drivers SHOULD always specify the ``nameOnly`` option when they only intend to
 access database names from the ``listDatabases`` command result.
 
+Drivers SHOULD support the ``filter`` option when implementing this method. 
```

Comment:
I'm not sure I see the use case for the filter option in list_database_names. To see if the names you specified actually exist?

Reply:
Perhaps you just want to list names with a common prefix (maybe relevant for multi-tenant apps).

Reply:
Ah, good point.